apps/docs/src/components/side-panel/side-panel.module.css

```diff
@@ -0,0 +1,8 @@
+.SidePanel {
+  position: relative;
+}
+
+.Header {
+  padding: var(--base-size-16);
+  border-bottom: 1px solid var(--borderColor-default, #d1d9e0);
```

Comment:
Close button is currently aligned to the start, can we move to end position instead?

```suggestion
  border-bottom: var(--brand-borderWidth-thin) solid var(--brand-color-border-muted);
  display: flex;
  justify-content: end;
```


<table>
<tr>
<th> Before</th> <th> After </th>
</tr>
<tr>
<td valign="top">

<!-- Insert "before" screenshot here -->
![Screenshot 2024-12-03 at 10 11 31](https://github.com/user-attachments/assets/454e9eed-977e-491c-8094-5635a93e00cb)

 </td>
<td valign="top">

<!-- Insert "after" screenshot here -->
![Screenshot 2024-12-03 at 10 11 25](https://github.com/user-attachments/assets/40d1138e-fd69-4c6e-9beb-e02b143bf48a)

</td>
</tr>
</table>

Reply:
Let me check what we do in PRC. I agree that the end position is much better.

Reply:
End position is the correct placement. I'm also not sure why the IconButton isn't using the `"invisible"` variant. I may update that as well.